flamedisx/er_nr_base.py

```diff
@@ -195,6 +198,16 @@ def electron_gain_mean(z, *, g2=20):
     photon_gain_std = 0.5
     double_pe_fraction = 0.219
 
+    # Dummy method for pax reconstruction bias mean.
+    # Implement it in source specific class. See x1t_sr1.py till exempel
+    def recon_bias_s1(self, sig, bias_pivot_pt=1.):
```

Comment:
sig -> s1? `bias_pivot_pt` is not used; users can add extra parameters in their overrides of this function